.github/workflows/release.yml

```diff
@@ -1,29 +1,79 @@
-# This Workflow performs a full release of chectl
-name: Release chectl
+#
+#  Copyright (c) 2012-2020 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+name: Build and Deploy release artifacts ⌛
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'The version that is going to be released. Should be in format 7.y.z'
-        required: true
-        default: '7.y.z'
+  # Trigger the workflow on push only for the master branch
+  push:
+    branches:
+      - '7*' # Push events to matching 7.* branches
+      - 'master'
 jobs:
-  build:
-    name: Create Release
-    runs-on: ubuntu-latest
+  build-and-deploy:
+    runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
         with:
-          fetch-depth: 0
-          token: ${{ secrets.ACTIONS_TOKEN }}
```

Comment:
why don't you need to fetch code using the token?

Reply:
Good question. I didn't change nothing. Comes from here: https://github.com/che-incubator/chectl/blob/remove_travis/.github/workflows/draft-release.yml#L14-L18

Reply:
ok